recipes/occt/meta.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: occt
+  version: 7.0.0
+
+source:
+  fn: occ.tar.gz
+  url: https://github.com/3drepo/occt/archive/d7355a4c5b7cac441e2c8821e369798266376a56.tar.gz
+  sha256: 1000486ada9dc48e766215c70e38982612780a9a7ccb113fd403aa207b8fabdc
+
+build:
+  number: 0
+  skip: true                              # [win and not py35]
```

Comment:
I think the style is to not align the selectors. Please fix this up in the feedstock